### ISSUE TITLE
"Export biomes as image" feature uses fixed Y-level, not user-configurable

### ISSUE THREAD

**Issue by FakenameBill** [Cubitect/cubiomes-viewer#125]
The new "Export biomes as image" feature does not allow the user to specify the Y-level of the biome map which is to be exported as an image.

No matter what Y-level is entered into the box to the left of the main map view when the "export biomes as image" option is selected, the resulting image is of the biome map at a fixed Y-level. There is no option in the "export biomes as image" settings to specify a different Y-level.

Through trial and error attempting to match the contours of underground biomes between the Cubiomes Viewer main map view and the exported images, I've determined that the baked-in Y level appears to be about 72.

**Comment by Cubitect** [comment]
Mmmh, it works for me...
The Y-level, the MC version, and whether it uses large biomes or not, should all be controlled by the current map settings at the top left, at the time of export (evaluated when you press "Ok").

**Comment by FakenameBill** [comment]
I just reproduced the issue. Here are the exact steps that I took:
1. Open the Windows version of Release 2.2.0 of Cubiomes Viewer
2. Set numeric seed to 0, Y-level to 300, version to 1.19
3. Confirm that the map window has updated correctly
4. Go to "Export biomes to image...", select 1:4 scale, change filename, enter custom coordinate bounds, hit OK
5. Resulting image appears to be at Y-level 72.

I also tried Y-level 256 and 250, both resulted in incorrect images; however, the the Y-level seems to not actually be fixed. The image created with the y-level set to 250 appears to actually represent Y-level 62, whereas the image at 300 represented 72.

Using Y-level 999, I was able to produce an image with no underground biomes.
